src/app/courses/add-courses/courses-add.component.ts

```diff
@@ -155,29 +170,46 @@ export class CoursesAddComponent implements OnInit {
     this.router.navigate([ '/courses' ]);
   }
 
+  isClassDay(day) {
+      return this.courseFrequency.includes(day) ? true : false;
+  }
+
+  updateCourseFrequency(courseFrequency, formArray) {
+    courseFrequency.map((c) => {
+      !formArray.value.includes(c) ? formArray.push( new FormControl(c)) : null ;
```

Comment:
map does not always require to return, so you can just use one if case not ternary operator